src/cv/Cv.js

```diff
@@ -48,100 +50,58 @@ class Cv extends Component {
     let experiencesList = [];
     let educationList = [];
     let skillsetList = [];
-    let headerLogo;
     let title;
+    let sourcesFirst = {};
```

Comment:
We don't need this variables here if we have page components. Instead of creating everything in this component and passing to pages as props we should move this code to pageFirst and pageSecond. It will give us a separated components.